gemini/sample-apps/fashion-trends-prediction/app/data_prep/driver.py

```diff
@@ -0,0 +1,332 @@
+import json
+import time
+
+from config import config
+from dataProcessing import getPosts
+from helper_functions_insta import getInfluencers, instalogin
+from helper_functions_vogue import get_articles
+from prepare_data_for_retriever import prepare_data_for_retriever
+
+parameters = config["parameters"]["standard"]
+country_names = config["countryList"]
+data_path = config["Data"]["current_data"]
+
+
+def isRoot(word):
```

Comment:
Is this function needed? With the refactoring suggested below, this would only need to be used once.